packages/java/parser-jvm-plugin-model/src/test/java/com/vaadin/fusion/parser/model/utils/TestBase.java

```diff
@@ -0,0 +1,46 @@
+package com.vaadin.fusion.parser.model.utils;
+
+import static com.vaadin.fusion.parser.testutils.OpenAPIAssertions.assertEquals;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.Path;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import com.vaadin.fusion.parser.core.Parser;
+import com.vaadin.fusion.parser.core.ParserConfig;
+import com.vaadin.fusion.parser.testutils.ResourceLoader;
+
+import io.swagger.v3.core.util.Json;
+import io.swagger.v3.oas.models.OpenAPI;
+
+public abstract class TestBase {
+    protected final ObjectMapper mapper = Json.mapper();
+    protected final ResourceLoader resourceLoader;
+    protected Path targetDir;
+
+    {
```

Comment:
could we move this into a @BeforeClass method? I think it's more clear than this instance initializer block.

Reply:
Nope, unfortunately. I don't understand why but JUnit fails to find a method with `@BeforeEach` in the parent class. If you know why it happens and how to fix that, it would be great to implement

Reply:
ok, I pushed a change here, at least locally the tests are fine.